README.md

```diff
@@ -1,7 +1,7 @@
 # System Requirements
  * Perl
- * DICOM toolkit
- * MINC tools
+ * DICOM toolkit (Will be installed by Install Script if not already installed)
+ * MINC tools (http://www.bic.mni.mcgill.ca/ServicesSoftware/MINC)
 
 Note: For Ubuntu installations, DICOM toolkit and MINC toolkit are installed by the imaging install script (see step 3 below). This script will apt-get install these packages: minc-tools and dcmtk.   
 
```

Comment:
@Jkat, 

I think this note should be updated (remove that the MINC toolkit will be installed).